internal/activePlot.go

```diff
@@ -150,8 +152,29 @@ func (ap *ActivePlot) RunPlot() {
 	if ap.Threads > 0 {
 		args = append(args, fmt.Sprintf("-r%d", ap.Threads))
 	}
+	if ap.PlotSize == 32 {
```

Comment:
if PlotSize is not set (ie. zero) then set it to 32, otherwise we should just let the user select whatever they want